### ISSUE TITLE
Train model with self dataset

### ISSUE THREAD

**Issue by CodingMice** [SystemErrorWang/White-box-Cartoonization#5]
Nice job! I train model with myself dataset. But I find that the results of pretrained model has a lot of noise. And it also happens in the final results.  I can not find the reason.

**Comment by CodingMice** [comment]
Otherwise, if I just cartoonization face image, can I use face segmetation inplace the  the Structure representation.

**Comment by SystemErrorWang** [comment]
In case of noise, I guess it was because of the guided filter layer. you can try to train a model without the guided filter in the generator and then add it when inference, you can also try to tune the parameters (r and eps). As for the face segmentation, i have no idea because I have not tried this method.

**Comment by zhiguangyang** [comment]
Noise always appear in the results of train.py, while in the results of cartoonize.py I could not find noise.

**Comment by itomorrower08** [comment]
@SystemErrorWang Nice job, but  when I used my own dataset to train, It has a bug in utils.py's function simple_superpixel 'batch_out = Parallel(n_jobs=num_job)(delayed(process_slic)\
                         (image) for image in batch_image)', I can not find the reason. Thankyou

**Comment by SystemErrorWang** [comment]
@itomorrower08 This may be because of environment conflict. In my own experiment, tensorflow=1.12.0/1.13.0, scikit-image==0.14.5 and joblib==0.13.0.

**Comment by itomorrower08** [comment]
@SystemErrorWang Thankyou, It's ok.  I also want to know whether the VGG19model that download from other places will also be ok? Because I cannot find the ''vgg19_no_fc.npy' in your git. And whether the training data  must be resized into 256*256 .

**Comment by SystemErrorWang** [comment]
@itomorrower08 
https://drive.google.com/open?id=1JfJzJbNjAWBIHGm9mc_R9dXv7DAw3tZc
you can download this file here

**Comment by itomorrower08** [comment]
@SystemErrorWang It's useful, thanks. By the way, how many training data(cartoon_photo and face_photo) do you uesd in this project?

**Comment by SystemErrorWang** [comment]
@itomorrower08 scenery photo：cyclegan training set；scenery cartoon：miyazaki hayao 3617 + hosoda mamoru 5107 + shinkai makoto 5891；face photo：ffhq 00000-10000；face cartoon：pa works 5000 + kyoto animation 5000

**Comment by Xinxiang7** [comment]
@SystemErrorWang may I ask what's the dataset of scenery cartoon look like? Does it only contains landscape? Or it contains buildings, food, streets, city, campus and many other topics?

**Comment by SystemErrorWang** [comment]
@Xinxiang7 we used the dataset provided in CycleGAN repository for scenery dataset. You can select the first 5000 images or use all 6227 images, this will only cause minor differeces.

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks very much. 
I mean the scenery cartoon? 
Does this contains different topics such as buildings, food, streets, city, campus and so on?

**Comment by SystemErrorWang** [comment]
@Xinxiang7 yes, it contains a wide variety of scenes and topics. for details of the dataset, please download and check it.

**Comment by Xinxiang7** [comment]
@SystemErrorWang Thanks Very much.

**Comment by Xinxiang7** [comment]
Hi, @zhiguangyang . May I have a look at your result. I tried different versions with different paratmeters, but the results are not that good.

**Comment by alexanderpoplavsky** [comment]
@SystemErrorWang Could you please share at least one exemplary picture for each training dataset group, as it is not clear from the description how to prepare the data properly. Thank you!

**Comment by SystemErrorWang** [comment]
@alexanderpoplavsky 
you can download the training set here, but it will be deleted if there're any copy rights issus
https://drive.google.com/file/d/10SGv_kbYhVLIC2hLlz2GBkHGAo0nec-3/view?usp=sharing

**Comment by alexanderpoplavsky** [comment]
@SystemErrorWang Thank you very much! I really appreciate your help!

**Comment by lilichu** [comment]
> @alexanderpoplavsky
> you can download the training set here, but it will be deleted if there're any copy rights issus
> https://drive.google.com/file/d/10SGv_kbYhVLIC2hLlz2GBkHGAo0nec-3/view?usp=sharing

hi, the dataset has some dirs, such as:
scenery cartoon：miyazaki hayao 3617 + hosoda mamoru 5107 + shinkai makoto 5891；
when you train, do you use only one dir or  all dirs?
